src/ResourceManagement/DataFactory/DataFactoryManagement/Customizations/Models/Datasets/StorageFormats/OrcFormat.cs

```diff
@@ -0,0 +1,24 @@
+﻿//
+// Copyright (c) Microsoft.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+namespace Microsoft.Azure.Management.DataFactories.Models
+{
+    /// <summary>
+    /// ORC format.
+    /// </summary>
+    public class OrcFormat : StorageFormat
```

Comment:
In Data Factory backend, it will take the OrcFormat's class name as the routine condition (a common practice for data factory formats). So it is necessary to create a empty "OrcFormat".